### DISCUSSION TITLE
Mixing this with Autogen

### DISCUSSION THREAD

**Discussion by schuerstedt** [microsoft/AI-Chat-App-Hack#29] [Ideas]
I got it running (really great script) with my own data. Needed some turns as a larger PDF with >1600 pages keeps crashing. 

While playing with the chat app, the idea comes up to use this as a kind of retrieval agent in AutoGen. 

Is anyone playing with this combination?

**Comment by zedhaque** [comment]
Just wondering what is the size of the 1600 page pdf file? Greater than 500 MB?

**Reply by schuerstedt** [reply]
1780 pages, 11MB size, pure text - tried to split with Adobe, but it complains >1500 pages :). Will print it out in parts as pdf and use this.

**Comment by pamelafox** [comment]
I haven't tried AutoGen but colleagues all say its great. I'd love to see how that works for you.
